src/dfeed/web/web/request.d

```diff
@@ -258,6 +258,9 @@ HttpResponse handleRequest(HttpRequest request, HttpServerConnection conn)
 				if (viewMode == "basic")
 					discussionGroup(groupInfo, page);
 				else
+				if (viewMode == "new")
```

Comment:
No, please don't call it "new". Use a descriptive name like `narrow-index`.

Reply:
No. All views attempt to be mobile-friendly to some extent. Even the horizontal-split view is usable on a smartphone to some extent (more so if you turn your phone horizontally).

Reply:
I don't see why you're opposed to this, narrow index is a terrible name for end users. What they need to know is that this view mode is specifically made for mobile screens so that they can easily decide to choose this view mode setting on their phone.

Reply:
I don't see why you're opposed to `narrow-index` considering we have modes that are called `horizontal-split` and `vertical-split`. By your logic, those should be called something like "classic", "comfortable" or "advanced", all of which are generic, non-descriptive and unhelpful.

Reply:
I also don't want to add a mode with a generic name because this is a highly opinionated take on the problem (as is any of them I suppose). I certainly don't mind submissions for highly opinionated view modes (it's why users have choice), and I'm not going to interfere with the stylistic choices taken in creating it (but I'm also probably not going to put much effort into maintaining it, because as I said on IRC and Discord I think this problem can also be addressed by improving the basic view mode). In principle, someone else may very well submit their own approach to solve the same problem, in which case it would be confusing and unfair to name their attempt `mobile-friendly-2` or something like that.

So, I also wouldn't be opposed to an unambiguously unique name like `ahmetsait-mobile`.

Reply:
Fair enough.